content/docs/02-guides/02-multi-modal-chatbot.mdx

```diff
@@ -204,11 +205,12 @@ export default function Chat() {
                 attachment?.contentType?.startsWith('image/'),
               )
               .map((attachment, index) => (
-                <img
+                <Image
                   key={`${m.id}-${index}`}
                   src={attachment.url}
                   width={500}
-                  alt={attachment.name}
+                  height={500}
```

Comment:
do we know the image sizes? since those are attachments? otherwise it'll distort